### ISSUE TITLE
Priority 2 Accessibility issue: Status Update 

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6738]
**What**

When page content updates there is no status announcement to inform users of the change.

Upon submitting a search term page content updates and text shows that there are ‘X results for [search term]’. Likewise, upon selecting a filter, text appears to inform users which Active filters have been applied. This is not announced to screen reader users and instead, users must navigate the page in order to determine that content has changed.

Solution:
Ensure that when page content updates a status message is announced to users informing them of the change. This can be accomplished by utilising an aria-live region using a role of ‘status’ on the container. Please note that the container must exist on the page prior to a change of content in order for the change to announce. This should be applied to the status messages that dynamically update, for example text showing ‘X results’ or ‘Active filters’.

**Where**

URL: https://wellcomecollection.org/works?query=Monkey Page: Monkey | Catalogue search | Wellcome 

<img width="964" alt="Screenshot 2021-07-07 at 14 50 09" src="https://user-images.githubusercontent.com/7315126/124770911-a1bfea00-df32-11eb-8ef4-318a69db0ef4.png">

**Comment by gestchild** [comment]
Fixed by #6722
